Backend/dev/migrations/dynamic-offer-driver-app/0000-db-init.sql

```diff
@@ -34,3 +34,107 @@ fare double precision NOT NULL
 ALTER TABLE atlas_driver_offer_bpp.fare_policy_per_extra_km_rate OWNER TO atlas_driver_offer_bpp_user;
 
 
+
+-- NOT FOR LOCAL HENCE COMMENTED --
```

Comment:
@ErasedSoul This Backup would not be required as we are only adding new columns.
```
ALTER TABLE atlas_driver_offer_bpp.merchant_push_notification ADD COLUMN id UUID DEFAULT md5(random()::text || clock_timestamp()::text)::uuid;
```
This you can remove, would be extra work. The index queries are good that we can run in Master and Prod.